tests/test_main.py

```diff
@@ -3,11 +3,11 @@
 from pathlib import Path
 from tempfile import TemporaryDirectory
 
-from calculadora_do_cidadao import Igpm, Inpc, Ipca, Ipca15, IpcaE, Selic
+from calculadora_do_cidadao import Igpm, Inpc, Ipca, Ipca15, IpcaE, Selic, Cpiaucsl
```

Comment:
Can we keep the alphabetical order in the `import`s? As the package grows bigger, these minor details can enhance maintainability ; )